x-pack/plugins/apm/public/components/app/correlations/correlations_table.tsx

```diff
@@ -7,65 +7,47 @@
 
 import React, { useCallback, useMemo, useState } from 'react';
 import { debounce } from 'lodash';
-import {
-  EuiIcon,
-  EuiLink,
-  EuiBasicTable,
-  EuiBasicTableColumn,
-  EuiToolTip,
-} from '@elastic/eui';
+import { EuiBasicTable, EuiBasicTableColumn } from '@elastic/eui';
 import { i18n } from '@kbn/i18n';
-import { useHistory } from 'react-router-dom';
-import { asInteger, asPercent } from '../../../../common/utils/formatters';
-import { APIReturnType } from '../../../services/rest/createCallApmApi';
 import { FETCH_STATUS } from '../../../hooks/use_fetcher';
-import { createHref, push } from '../../shared/Links/url_helpers';
-import { ImpactBar } from '../../shared/ImpactBar';
 import { useUiTracker } from '../../../../../observability/public';
 import { useTheme } from '../../../hooks/use_theme';
+import type { SelectedSignificantTerm } from '../../../../common/search_strategies/correlations/types';
 
+export type { SelectedSignificantTerm };
```

Comment:
Not sure this is necessary? Can the other places that use this type also import from `common`?

Reply:
Fixed here https://github.com/elastic/kibana/pull/108441/commits/f4ecc6642b103d7d1828a4ec07540fa0f138ca47